### PR TITLE
Fixes #18457: Typo in API doc

### PR DESCRIPTION
**Pull request by amousset**
https://issues.rudder.io/issues/18457

### CODE REVIEW

File: webapp/sources/api-doc/paths/nodes/apply-policy.yml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # SPDX-License-Identifier: CC-BY-SA-2.0
 # SPDX-FileCopyrightText: 2013-2020 Normation SAS
 post:
-  summary: Trigger a agent runs
-  description: This API allows to trigger an agent run on the target node. Response contains a json stating if agent could be started on each node, but not if the run went fine and do not display any output from it. You can see the result of the run in Rudder web interface or in the each agent logs.
+  summary: Trigger an agent runs on all nodes
```
Comment (Fdall) [inline webapp/sources/api-doc/paths/nodes/apply-policy.yml:4]:
```suggestion
  summary: Trigger an agent run on all nodes
```

**Comment by amousset** [comment]
PR updated with a new commit

File: webapp/sources/api-doc/paths/nodes/apply-policy.yml (hunk 1 above)
Comment (amousset) [reply webapp/sources/api-doc/paths/nodes/apply-policy.yml]: bien vu

**Review by Fdall — APPROVED** [review]

**Comment by amousset** [comment]
OK, squash merging this PR

**Comment by amousset** [comment]
OK, merging this PR

**Pull request merged**